src/elements/kw-share-shell/demo/index.html

```diff
@@ -0,0 +1,218 @@
+                <kw-share-shell id="share" share>
+                    <div slot="share-card">
+                        <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade tombstone$="[[!share]]"></iron-image>
+                        <div class="bottom">
+                            <div class="stats">
+                                <div class="likes">
+                                    <span><img src="assets/icons/shares/like.svg"> </img>100 likes</span>
+                                </div>
+                                <div class="comments">
+                                    <span><img src="assets/icons/shares/comment_b.svg"> </img>300 comments</span>
+                                </div>
+                                <div class="shares">
+                                    <span><img src="assets/icons/shares/share.svg"> </img>share</span>
+                                </div>
+                            </div>
+                            <div class="line"></div>
+                            <div class="content">
+                                <div class="description">
+                                    <h3></h3>
+                                    <p></p>
+                                </div>
+                                <div class="actions">
+                                    <button type="submit" class="orangebutton remixbutton" onclick="">remix<img src="assets/icons/shares/remix.svg" /></button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </kw-share-shell>
+                <h3>with share data</h3>
+                <kw-share-shell id="detail">
+                    <div slot="share-card">
+                        <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade tombstone$="[[!share]]"></iron-image>
+                            <div class="bottom">
+                                <div class="stats">
+                                    <div class="likes">
+                                        <span><img src="assets/icons/shares/like.svg"> </img>66 likes</span>
+                                    </div>
+                                    <div class="comments">
+                                        <span><img src="assets/icons/shares/comment_b.svg"> </img>100 comments</span>
+                                    </div>
+                                    <div class="shares">
+                                        <span><img src="assets/icons/shares/share.svg"> </img>share</span>
+                                    </div>
+                                </div>
+                                <div class="line"></div>
+                                <div class="content">
+                                    <div class="description">
+                                        <h3>My awesome share</h3>
```

Comment:
the hard-coded second half of the demo is a bit misleading I think, not sure if necessary?